cds-snc/cppd-medical-report-sails
# Declaration ma view

# What this does
Adds the Declaration view for the Adjudicators report view.

Replaces #159 

This does not do all of what #159 did (ie, capturing timezone). This is a stripped-down mvp. Will keep that PR around and circle back later.

Also did a bit of refactoring moving the report consent view to a separate controller (and matching report/declaration controller).

## To test
- Create a new report, fill it out and submit the patient and practitioner portions.
- Login as an Adjudicator, select your report
- Click the link to view the Declaration

Should look like this:
![image](https://user-images.githubusercontent.com/1187115/76656071-35eb8e00-6545-11ea-8dff-db1c73b224f3.png)
